src/Foundatio.Extensions.Hosting/Cronos/CalendarHelper.cs

```diff
@@ -5,7 +5,7 @@
 using System;
 using System.Runtime.CompilerServices;
 
-namespace Cronos;
```

Comment:
Not sure why we never did this, but would be nice to pull something like this in to help us upgrade this lib. https://github.com/exceptionless/Exceptionless.Net/blob/main/src/Exceptionless/update-json.ps1

Reply:
It's only like 5 files and isn't likely to need to be updated.